examples/prompt/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/bitfield/script"
+)
+
+func main() {
+	script.Echo("Installation of Fake Product\n").Stdout()
+	curDir, err := script.Exec("pwd").String()
+	if err != nil {
+		panic(err)
+	}
+
+	script.Prompt("Choose install location", strings.TrimSpace(curDir)).Stdout()
```

Comment:
Suppose you use the install location as an argument to a fake `install` command, for example?